prow/cluster/prowjob_service_accounts.yaml

```diff
@@ -1,13 +1,23 @@
 ---
 apiVersion: v1
 kind: ServiceAccount
+metadata:
+  annotations:
+    iam.gke.io/gcp-service-account: istio-prow-test-job-default@istio-testing.iam.gserviceaccount.com
+  labels:
+    app.kubernetes.io/part-of: prow
+  namespace: test-pods
+  name: prowjob-default-sa
+---
+apiVersion: v1
+kind: ServiceAccount
 metadata:
   annotations:
     iam.gke.io/gcp-service-account: istio-prow-test-job@istio-testing.iam.gserviceaccount.com
   labels:
     app.kubernetes.io/part-of: prow
   namespace: test-pods
-  name: prowjob-admin-sa
+  name: prowjob-advanced-sa
```

Comment:
Can we comment somewhere what permissions this actually has? Its pretty hard to tell from this which will likely lead to incorrect usage

Reply:
Done in another PR.